include/UIKit/UIApplicationDelegate.h

```diff
@@ -129,6 +131,7 @@ UIKIT_EXPORT NSString* const UIApplicationLaunchOptionsToastActionUserInputKey;
 - (void)application:(UIApplication*)application didReceiveRemoteNotification:(NSDictionary*)userInfo;
 - (void)application:(UIApplication*)application didReceiveToastAction:(NSDictionary*)action;
 - (void)application:(UIApplication*)application didReceiveVoiceCommand:(WMSSpeechRecognitionResult*)result;
+- (void)application:(UIApplication*)application didReceiveFile:(WAAFileActivatedEventArgs*)result;
```

Comment:
it seems they are different patterns with the ability for multiple files and Verbs (open vs post vs edit, etc) and the ability to be called while active as Nithish points out. In light of that, I think the pattern here should be used rather than reducing fidelity / trying to cram extra parameters into an options dictionary that probably wouldn't be checked and be incongruous with the other delegate call.

---
In reply to: [82463028](https://github.com/Microsoft/WinObjC/pull/1100#discussion_r82463028) [](ancestors = 82463028,82305070)